src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -10,120 +10,122 @@ The following example PET datasets have been formatted using this specification
-Examples are `acq-18FFDG` for fludeoxyglucose, `acq-11CPIB` for Pittsburgh compound B, etc.
-
-**Reconstruction:** The reconstruction key (`recon-<label>`) has four reserved values:
-- `acdyn`, for reconstructions with attenuation correction of dynamic data;
-- `acstat`, for reconstructions with attenuation correction of static data;
-- `nacdyn`, for reconstructions without attenuation correction of dynamic data;
-- `nacstat`, for reconstructions without attenuation correction of static data.
+In this example, tracer injection coincides with scan start.
+
+## PET recording data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    pet/
+        sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.nii[.gz]
+        sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.json
+```
+
+PET data belong in the `pet` directory.
+PET imaging data SHOULD be stored in 4D (or 3D, if only one volume was acquired)
+NIfTI files with the `_pet` suffix.
+Volumes MUST be stored in chronological order (the order they were acquired in).
+
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) is used to
+indicate a task subjects were asked to perform in the scanner.
+Those labels MUST be consistent across subjects and sessions.
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity is used to
+indicate the tracer used.
+This entity is OPTIONAL if only one tracer is used in the study,
+but REQUIRED to distinguish between tracers if multiple are used.
+The label used is arbitrary and each file requires a separate JSON sidecar
+with details of the tracer used (see below).
+Examples are `acq-18FFDG` for fludeoxyglucose or `acq-11CPIB` for Pittsburgh compound B.
+Other labels are permitted, as long as they are consistent across subjects and sessions
+and consist only of the legal label characters.
+
+If more than one run of the same task and acquisition (tracer) are acquired during
+the same session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used:
+`_run-1`, `_run-2`, `_run-3`, and so on.
+If only one run was acquired the `run-<index>` can be omitted.
+
+The OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec) entity
+is used to indicate the reconstruction method used for the image,
+with four reserved values:
+
+  - `acdyn`, for reconstructions with attenuation correction of dynamic data;
+  - `acstat`, for reconstructions with attenuation correction of static data;
+  - `nacdyn`, for reconstructions without attenuation correction of dynamic data;
+  - `nacstat`, for reconstructions without attenuation correction of static data.
+
 Further details regarding reconstruction are in the `_pet.json` file.
-In case of multiple reconstructions of the data with the same type,
-we allow for using a number after the `<label>` in order to distinguish,
+If multiple reconstructions of the data are made with the same type,
```

Comment:
```suggestion
If multiple reconstructions of the data are made with the same type of reconstruction,
```

Reply:
@sappelhoff. I made som additional changes. I would like to wait for others to review it before merging. @melanieganz

Reply:
Applied.